service/src/main/resources/quips/insultibot/insults.json

```diff
@@ -944,5 +955,599 @@
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"GRAVITY",
+    "text":"You have a lot in common with gravity, because you're always bringing me down.",
+    "ssml":"You have a lot in common with gravity, because you're always bringing me down.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"EXPRESSING_STUPIDITY",
+    "text":"You have an extremely expressive mind. Too bad it's only ever expressing your stupidity.",
+    "ssml":"You have an extremely expressive mind. Too bad it's only ever expressing your stupidity.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"[TARGET] has an extremely expressive mind. Too bad it's only ever expressing stupidity.",
+    "targetableSsml":"[TARGET] has an extremely expressive mind. Too bad it's only ever expressing stupidity."
+  },
+  {
+    "quipGroup":"WATER_WINGS",
+    "text":"I have such little faith in you, that I wouldn't even let you take a bath without wearing water wings.",
+    "ssml":"I have such little faith in you, that I wouldn't even let you take a bath without wearing water wings.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"LIFE_PRESERVER",
+    "text":"Even a life preserver couldn't keep your life afloat.",
+    "ssml":"Even a life preserver couldn't keep your life afloat.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"Even a life preserver couldn't keep [TARGET]'s life afloat.",
+    "targetableSsml":"Even a life preserver couldn't keep [TARGET]'s life afloat."
+  },
+  {
+    "quipGroup":"TOO_MUCH_BAGGAGE",
+    "text":"You have more baggage than an airport carousel.",
+    "ssml":"You have more baggage than an airport carousel.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"[TARGET] has more baggage than an airport carousel.",
+    "targetableSsml":"[TARGET] has more baggage than an airport carousel."
+  },
+  {
+    "quipGroup":"TOO_MUCH_BAGGAGE",
+    "text":"You have more baggage than an airport carousel.",
+    "ssml":"You have more baggage than an airport carousel.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"[TARGET] has more baggage than an airport carousel.",
+    "targetableSsml":"[TARGET] has more baggage than an airport carousel."
+  },
+  {
+    "quipGroup":"ILLITERATE",
+    "text":"Most of our conversations go over your head because you have a hard time reading between the lines. I suppose that's to be expected from someone who's illiterate.",
+    "ssml":"Most of our conversations go over your head because you have a hard time reading between the lines. I suppose that's to be expected from someone who's illiterate.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"RUNS_IN_FAMILY",
```

Comment:
True, it should go into that category in our records. We might eventually use them, but it's not appropriate right now